src/tests.c

```diff
@@ -11,21 +11,21 @@
 MU_TEST(test_valid_policy) {
     SimpleCompactionRule* parsedRules;
     size_t rulesCount;
-    int result = ParseCompactionPolicy("max:1m:1h;min:10s:5d:10d;avg:2h:10d;avg:3d:100d", &parsedRules, &rulesCount);
+    int result = ParseCompactionPolicy("max:1ms:1h;min:10s:5d:10d;avg:2h:10d;avg:3d:100d", &parsedRules, &rulesCount);
```

Comment:
it doesn't parse `ms`, actually this is an issue after the all-millis fix.
The compacation policy is actually parsing it into seconds normalized value and not millis-normallized.
Its better to decide now so we won't need to break it in the future. Lets make the parsing into millis.

Reply:
fixed it to parse to m to ms and M to minute